modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -600,6 +600,15 @@ CV__DNN_INLINE_NS_BEGIN
         static Ptr<RegionLayer> create(const LayerParams& params);
     };
 
+    /**
+     * @brief \f$ L_p \f$ - detection output layer.
```

Comment:
What is Lp?

Reply:
Oh. I copied the comment from below and left that part. Removing it now.